portfolio/src/main/webapp/mapsScript.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+function initMap() {
+    var coordin_wat = {lat: 43.4643, lng: -80.5204};
+    var coordin_tor = {lat:43.6532, lng: -79.3832};
```

Comment:
As per the style guide (https://google.github.io/styleguide/jsguide.html#features-use-const-and-let), use const wherever possible; use let if you need to reassign the variable; never use var.
Naming: https://google.github.io/styleguide/jsguide.html#naming
Avoid using abbreviations (wat and tor are meaningless for people outside of this area). Use camelCase for variables and CONSTANT_CASE for constants (e.g. "var coordin_wat" should be "const COORDINATE_WATERLOO" (unless the struct is called "coordin")).

Reply:
Fixed!